browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1456,7 +1456,7 @@ Are you sure you want to do this?
         Privacy Policy
       </message>
       <message name="IDS_TAKE_QUICK_TOUR" desc="Brave rewards quick tour text">
-        Take a quick tour
+        Let’s take a quick tour
```

Comment:
Nit: should this be an apostrophe ([`U+0027`](https://www.fileformat.info/info/unicode/char/0027/index.htm)), instead of a right single quotation mark ([`U+2019`](https://www.fileformat.info/info/unicode/char/2019/index.htm))? We've been having this issue in pretty much all our `GRD` files. I think we should stick with one of them.

Reply:
i have seen many `apostrophe` in existing strings, so i kept it like it.

Reply:
Sure, but this is a right single quotation mark, not an apostrophe. 🙂
But this we can address separately, as it's a global problem.